accounts-db/src/append_vec.rs

```diff
@@ -367,6 +402,7 @@ impl AppendVec {
                 }
                 Ok(())
             }
+            AppendVecFileBacking::FileOnly(_file) => Ok(()),
```

Comment:
If this is because FileOnly is only used for readonly mode, I think we should add a comment that
says so

Reply:
done